### PR TITLE
FOGL:3193: Inconsistent timezone issue fix

### PR DESCRIPTION
**Pull request by mshariq-nerd**
1. Select UTC and local timezone option added in settings page
2. By default all timestamps converts into local timezone in the GUI 

Logged [FOGL-6083](https://scaledb.atlassian.net/browse/FOGL-6083) for API changes to make all api calls response in UTC timezone

### CODE REVIEW

File: src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts [hunk 1]
```diff
@@ -215,7 +215,7 @@ export class ReadingsGraphComponent implements OnDestroy {
     const numReadings = [];
     const strReadings = [];
     const arrReadings = [];
-    this.timestamps = readings.map((r: any) => r.timestamp);
+    this.timestamps = readings.reverse().map((r: any) => r.timestamp);
```
Comment (praveen-garg) [inline src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts:218]: will this map okay, after reversing?

File: src/app/components/core/backup-restore/backup-restore.component.html [hunk 2]
```diff
@@ -72,7 +72,7 @@
             <tbody>
               <tr *ngFor="let backup of backupData">
                 <td class="align-content">
-                  <small>{{ backup.date | dateparser:"YYYY-MM-DD hh:mm:ss"}}</small>
+                  <small>{{ backup.date | dateparser:"YYYY-MM-DD HH:mm:ss"}}</small>
```
Comment (praveen-garg) [inline src/app/components/core/backup-restore/backup-restore.component.html:75]: were we showing incorrectly by transforming to 12hour format?

File: src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts (hunk 1 above)
Comment (mshariq-nerd) [reply src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts:218]: yes

File: src/app/components/core/backup-restore/backup-restore.component.html (hunk 2 above)
Comment (mshariq-nerd) [reply src/app/components/core/backup-restore/backup-restore.component.html:75]: yes, it was in 12hour format

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**